src/main/target/SPRACINGF3NEO/target.c

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
Please align timer definitions with new timer framework - see other targets for how it's done now.